### PR TITLE
Tiny fixups to support #1548

### PR DESCRIPTION
**Pull request by joshuatcasey**
Tiny fixups to support #1548

### CODE REVIEW

File: internal/certauthority/certauthority_test.go [hunk 1]
```diff
@@ -20,60 +20,65 @@ import (
...
-	certPEM, err := os.ReadFile(certPath)
-	require.NoError(t, err)
-
-	keyPEM, err := os.ReadFile(keyPath)
-	require.NoError(t, err)
-
-	ca, err := Load(string(certPEM), string(keyPEM))
-	return ca, err
-}
+var (
```
Comment (benjaminapetersen) [inline internal/certauthority/certauthority_test.go:23]: Cool this is a nice little cleanup.

File: internal/controller/impersonatorconfig/impersonator_config.go [hunk 2]
```diff
@@ -363,6 +379,7 @@ func (c *impersonatorConfigController) serviceExists(serviceName string) (bool,
 		return false, nil, nil
 	}
 	if err != nil {
+		// untested
```
Comment (benjaminapetersen) [inline internal/controller/impersonatorconfig/impersonator_config.go:382]: Do we want all of these "untested" comments to linger in the production code?

File: internal/controller/impersonatorconfig/impersonator_config.go [hunk 3]
```diff
@@ -313,6 +312,23 @@ func (c *impersonatorConfigController) doSync(syncCtx controllerlib.Context, cre
 	return credentialIssuerStrategyResult, nil
 }
 
+func (c *impersonatorConfigController) ensureCAAndTLSSecrets(ctx context.Context, nameInfo *certNameInfo) (impersonationCABundle []byte, err error) {
```
Comment (cfryanr) [inline internal/controller/impersonatorconfig/impersonator_config.go:315]: I don't think you want to name the return types here, do you?

File: internal/controller/impersonatorconfig/impersonator_config.go (hunk 2 above)
Comment (cfryanr) [reply internal/controller/impersonatorconfig/impersonator_config.go:382]: Same question. If we were to put that comment on every line of untested code in the project, that would be a lot of distracting comments. :)

Comment (joshuatcasey) [reply internal/controller/impersonatorconfig/impersonator_config.go:382]: I'll remove with the next push

File: internal/controller/impersonatorconfig/impersonator_config.go (hunk 3 above)
Comment (joshuatcasey) [reply internal/controller/impersonatorconfig/impersonator_config.go:315]: I'll remove with the next push

**Review by cfryanr — APPROVED** [review]

**Pull request merged**